internal/cmd/db.go

```diff
@@ -121,22 +121,25 @@ var dbCmd = &cobra.Command{
 
 const ENV_ACCESS_TOKEN = "TURSO_API_TOKEN"
 
-func getAccessToken() (string, error) {
+func getAccessToken(warnMultipleAccessTokenSources bool) (token string, err error) {
 	envToken := os.Getenv(ENV_ACCESS_TOKEN)
-	if envToken != "" {
-		return envToken, nil
-	}
 	settings, err := settings.ReadSettings()
 	if err != nil {
 		return "", fmt.Errorf("could not read local settings")
 	}
+	settingsToken := settings.GetToken()
 
-	token := settings.GetToken()
-	if !isJwtTokenValid(token) {
+	if envToken != "" && settingsToken != "" && warnMultipleAccessTokenSources {
+		fmt.Println("Warning: found both environment and configuration token. Proceeding with environment one")
```

Comment:
I think we could improve the the error message to be more user friendly. Something like:
```
User logged in as <fetch user name from settings> but TURSO_API_TOKEN environment variable set so using it instead.
```